datascience/maps.py

```diff
@@ -0,0 +1,510 @@
+"""
+
+Integrates the folium package to draw maps.
+
+"""
+
+from IPython.core.display import HTML
+import json
+import folium
+import random
+
+#########################
+# Exposed Functionality #
+#########################
+
+
+def draw_map(center=None, **kwargs):
+    """ 
+    Draw a map with center & zoom containing all points and
+    regions that displays points as circles and regions as polygons.
+    """
```

Comment:
For someone who's using `help(draw_map)`, the info that'll be displayed here isn't super helpful. What can I pass in to `**kwargs`? Just looking at the method signature itself and the docstring doesn't tell the user what to call `draw_map` with, so we should list out the argument list explicitly either in the method sig itself or in the docstring.

(This is one of the reasons why I somewhat prefer explicitly listing the arguments; it's more verbose but also more helpful to someone looking at it for the first time.)